### PR TITLE
add phoenix observability cookbook

### PR DESCRIPTION
**Pull request by andrewrreed**
# What does this PR do?

Fixes # (issue)

## Who can review?

Feel free to tag members/contributors who may be interested in your PR.
@merveenoyan @stevhliu

### CODE REVIEW

File: notebooks/en/phoenix_observability_on_hf_spaces.ipynb [hunk 1]
```diff
@@ -0,0 +1,594 @@
+{
```
Comment (merveenoyan) [inline notebooks/en/phoenix_observability_on_hf_spaces.ipynb:1]: <p>would be nice to show a screenshot or two about the features IMO </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (stevhliu) [inline notebooks/en/phoenix_observability_on_hf_spaces.ipynb:1]: <p>The "Note" can be replaced with <code>&gt; [!TIP]</code> to use the doc-builder's syntax (same for the Note above)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Review by stevhliu — APPROVED** [review]
Really cool, thanks!

**Comment by MoritzLaurer** [comment]
Very cool notebook @andrewrreed ! 

One issue: when I tried to install the crew-ai dependencies, I got this error: 
Screenshot 2024-11-04 at 11 42 42

**Comment by MoritzLaurer** [comment]
It would be good to add a note that users should restart their environment after they install the dependencies for the crew AI example. These new dependencies install some underlying library versions that are only active after an kernel restart, which leads to errors if users just run the notebook sequentially without a restart (at least in my jupyterlab CPU environment). 

Two errors I got when not doing a kernel restart: 
```py
AttributeError: `np.float_` was removed in the NumPy 2.0 release. Use `np.float64` instead.
```
```py
FileNotFoundError: [Errno 2] No such file or directory: '[/home/user/miniconda/lib/python3.10/site-packages/jiter-0.7.0.dist-info/METADATA](https://moritzlaurer-jupyterlab.hf.space/home/user/miniconda/lib/python3.10/site-packages/jiter-0.7.0.dist-info/METADATA)'
```

File: notebooks/en/phoenix_observability_on_hf_spaces.ipynb (hunk 1 above)
Comment (andrewrreed) [reply notebooks/en/phoenix_observability_on_hf_spaces.ipynb:1]: <p>Agreed! just added</p>

Comment (andrewrreed) [reply notebooks/en/phoenix_observability_on_hf_spaces.ipynb:1]: <p>Good call, thanks! Just updated all of them</p>

**Pull request merged**